### DISCUSSION TITLE
usbipd-win not working with Segger JLink Debugger

### DISCUSSION THREAD

**Discussion by Parker-McDonnell** [dorssel/usbipd-win#162] [Q&A] [answered]
Hello,

First time posting to Github so hopefully this is the correct place.  I've been trying to attach my JLink debugger to Ubuntu 20.04 LTS running within WSL2.  I can attach and use other USB devices just fine, but the JLIink pod doesn't seem to work (aka I get a "[  JLink] - Cannot connect to J-Link." error message). 

I know other people are having the same problem.  Could be a driver permissions issues.

Thanks,
Parker

**Comment by dorssel** [comment, marked as answer]
Looks like a WSL-side issue. Does this comment help? https://github.com/dorssel/usbipd-win/issues/96#issuecomment-992804504

**Reply by Parker-McDonnell** [reply]
Got it.  Yes, I just talked with another developer and they supposedly have their JLink pod working on a custom WSL2 Ubuntu kernel (enables all driver permissions by default).  Likely I'm having a permissions issue.  Thanks!  I'll reply here once I get it working.

**Reply by Parker-McDonnell** [reply]
Sorry somehow missed your link!  Yes restarting the udev service fixed the problem!!!

```
sudo service udev restart
sudo udevadm control --reload
```

Thanks for the help!
